src/server/req_manager.c

```diff
@@ -178,6 +188,16 @@ extern int delete_attr_db(pbs_db_conn_t *conn,
 	struct svrattrl *pal);
 #endif /* localmod 005 */
 
+/*
+ * This structure used as part of the avl tree 
+ * to do a faster lookup of hostnames.
+ * It is stored against pname in make_host_addresses_list()
+ */
+struct pul_store {
```

Comment:
Sure, but I can't figure out what to name it.  Other things in the PBS code are already too similar (but for different purposes).  Can you provide a suggestion?